tilavarauspalvelu/models/reservation_unit_pricing/actions.py

```diff
@@ -32,4 +33,4 @@ def calculate_reservation_price(self, duration: datetime.timedelta, *, subsidise
         duration_seconds = int(duration.total_seconds())
         duration_minutes = int(math.ceil(duration_seconds / 60 / 15) * 15)
 
-        return (price / price_unit.in_minutes) * duration_minutes
+        return round_decimal((price / price_unit.in_minutes) * duration_minutes, 2)
```

Comment:
I don't think rounding should be done to the price here, it can be applied on relevant precision when needed, but this function should return the _actual_ price.